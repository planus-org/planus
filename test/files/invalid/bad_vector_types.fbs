table Foo {
  x: [[Bar]];
  y: [[Foo: 1]];
  z: [[Foo]];
}

union Bar {}
