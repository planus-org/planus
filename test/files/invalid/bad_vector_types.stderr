error: Unsupported type
  ┌─ test/files/invalid/bad_vector_types.fbs:2:7
  │
2 │   x: [[Bar]];
  │       ^^^^^ Vectors in vectors are not currently supported

error: Unsupported type
  ┌─ test/files/invalid/bad_vector_types.fbs:3:7
  │
3 │   y: [[Foo: 1]];
  │       ^^^^^^^^ Arrays in vectors are not currently supported

error: Unsupported type
  ┌─ test/files/invalid/bad_vector_types.fbs:4:7
  │
4 │   z: [[Foo]];
  │       ^^^^^ Vectors in vectors are not currently supported

